### ISSUE TITLE
documentation error

### ISSUE THREAD

**Issue by dasturge** [alectio/AlectioSDK#76]
spec for the user defined test funciton states predictions and labels should be `a dictionary of an index and a prediction/label for each test sample`, however the code calls `sklearn.metrics.confusion_matrix` and expects them to be lists/arraylike.

**Comment by deviprasad97** [comment]
Thank you @dasturge for pointing that out. We have updated our documentation to reflect that.
[commit](https://github.com/alectio/AlectioSDK/commit/dd9d61ee321610da9fd5396668da7fa881c5a0ae)
